### ISSUE TITLE
Most sites doesn't work anymore

### ISSUE THREAD

**Issue by Mantve** [sin3hz/Mbooru#14]
Don't know why, but all sites (except yandere) suddently doesn't work anymore. Probably they have updated the image board or something.
This also happens for other devices, so this must be the app issue.

**Comment by vtorresortiz** [comment]
I love this app but I don't find the code to submit a patch. I find this app and I fix it

https://github.com/vtorresortiz/danbooru-gallery-android/raw/master/DanbooruGallery/DanbooruGallery-release.apk

**Comment by vgaparadise** [comment]
Yeah, so many people love this app. If ur not going to work on it, let other people help. :(
Pick a license you like.
